.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock', '.yarnclean') }}
 
       - name: Install dependencies
-        run: yarn install && yarn bootstrap
+        run: yarn install && yarn bootstrap && yarn orbit-design-tokens build
```

Comment:
Better add another step below named `Build design tokens` for now, just to be explicit about it, later we will work on adding more sense to our scripts.